ffmuc-mesh-vpn-wireguard-vxlan/shsrc/lib/gluon/gluon-mesh-wireguard-vxlan/checkuplink

```diff
@@ -11,6 +11,12 @@ if [[ "${mesh_vpn_enabled}" == "0" ]]; then
 	exit 0
 fi
 
+# Some legacy code seem to have used "true" instead of the canonical "1".
+# This should be overwritten by a gluon-reconfigure (see 400-mesh-vpn-wireguard)
+if [[ "${mesh_vpn_enabled}" != "1" ]]; then
```

Comment:
I think it would be better to do this after checking for the lock. So we don't spam the log and only write the message when the rest of the code is executed

Reply:
so maybe we should move https://github.com/T0biii/community-packages/blob/patch-1/ffmuc-mesh-vpn-wireguard-vxlan/shsrc/lib/gluon/gluon-mesh-wireguard-vxlan/checkuplink#L7-L18 under the lock check

Reply:
Then we would create lockfileS (more writes) even if checkuplink is disabled.